README.md

```diff
@@ -90,7 +90,7 @@ Username: fossy
 Password: fossy
 ```
 
-**Note:** The test instance is not up to date with the latest release. The instance will reset every night at 2 am UTC and all the user uploaded data will be lost.
+**Note:** The test instance is not up to date with the latest release. The instance will reset every night at 2 am UTC and all the users' uploaded data will be lost.
```

Comment:
not quite convinced by the change, I think "user uploaded data" was OK as "user uploaded" kind of acts like an adjective for "uploaded data".
However, I would make it even simpler by changing to "all uploaded data will be lost." :-)

Reply:
yaa,it will be fine